From ff5b36350424dae30b84f6e258e0be4a7d023f5c Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Fri, 29 Mar 2019 11:43:30 +0700
Subject: [PATCH 150/175] pinctrl: sh-pfc: r8a77990: fix size of pin group
 common part array

This patch fixes size of pin group common part array.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 drivers/pinctrl/sh-pfc/pfc-r8a77990.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pinctrl/sh-pfc/pfc-r8a77990.c b/drivers/pinctrl/sh-pfc/pfc-r8a77990.c
index 9d136ce..7b8ff3b 100644
--- a/drivers/pinctrl/sh-pfc/pfc-r8a77990.c
+++ b/drivers/pinctrl/sh-pfc/pfc-r8a77990.c
@@ -3038,7 +3038,7 @@ static const unsigned int du_disp_mux[] = {
 };
 
 static const struct {
-	struct sh_pfc_pin_group common[184];
+	struct sh_pfc_pin_group common[186];
 	struct sh_pfc_pin_group r8a77990[0];
 } pinmux_groups = {
 	.common = {
-- 
2.7.4

